plugins/gestures.koplugin/defaults.lua

```diff
@@ -76,6 +84,14 @@ return {
         hold_top_right_corner = nil,
         hold_bottom_left_corner = nil,
         hold_bottom_right_corner = nil,
+        one_finger_swipe_east = nil,
+        one_finger_swipe_west = nil,
```

Comment:
The default values here should be:

```lua
one_finger_swipe_east = {page_jmp = -1,},
one_finger_swipe_west = {page_jmp = 1,},
```

But that creates an extra page forward/back on top of what reading rolling/paging already does. Any ideas as to how to deal with this?

Reply:
Probably much like the full-refresh itself: by getting rid of the RR/RP handler.

Reply:
Just realized this PR now definitely conflicts with https://github.com/koreader/koreader/pull/7572 since `Disable swipes` is rendered redundant by the ability to set it to `Nothing` in the Gesture Manager. At the same time though, the page turns behavior should probably be independent from plugins. No idea what would be best.